CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
```

Comment:
The changelog doesn't reflect the released versions contained here, is that on purpose?

Reply:
Yeah, since 1.16.0 isn't build on 1.15.x, the changelog reflects the divergence. The changelog entries should be in place, at least in so far as they make sense in the given branch.

Reply:
I think the merge resolution should instead reflect the release versions for the features in this list. Some of these items are in fact unreleased, but the others were released with 1.15.1 and to be released with 1.15.2.